fs/base.py

```diff
@@ -1648,8 +1648,8 @@ def check(self):
         if self.isclosed():
             raise errors.FilesystemClosed()
 
-    def match(self, patterns, name):
-        # type: (Optional[Iterable[Text]], Text) -> bool
+    def match(self, patterns, name, accept_prefix=False):
```

Comment:
Should `accept_prefix` be added to the "Arguments:" docstring here?

Reply:
Or was the addition of this extra parameter an error, as it doesn't actually seem to be used in this function?